deserializers/src/main/java/com/nextdoor/bender/deserializer/json/JsonPathProvider.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * Copyright 2017 Nextdoor.com, Inc
+ *
+ */
+
+package com.nextdoor.bender.deserializer.json;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.Option;
+import com.jayway.jsonpath.Predicate;
+import com.jayway.jsonpath.spi.json.GsonJsonProvider;
+import com.jayway.jsonpath.spi.json.JsonProvider;
+import com.jayway.jsonpath.spi.mapper.GsonMappingProvider;
+import com.jayway.jsonpath.spi.mapper.MappingProvider;
+
+/**
+ * Statically initializes JsonPath and provides commonly used methods of JsonPath
+ */
+public class JsonPathProvider {
+  /**
+   * Setup JsonPath defaults
+   */
+  static {
+    /*
+     * Set static configuration for JsonPath
+     */
+    com.jayway.jsonpath.Configuration.setDefaults(new com.jayway.jsonpath.Configuration.Defaults() {
+      private final JsonProvider jsonProvider = new GsonJsonProvider();
```

Comment:
Context on this is that the `JsonPath` needs to be statically configured once and only once. Wrapping some of the functionality like this ensures it's configured on first access from anywhere in the code.